modules/nodes-adding-bare-metal-host-to-cluster-using-web-console.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-adding-bare-metal-host-to-cluster-using-web-console_{context}"]
+= Adding a bare metal host to the cluster using the web console
+
+You can add bare metal hosts to the cluster in the web console.
+
+.Prerequisites
+
+* Install a cluster on bare metal, and provision a new bare metal host for use in the cluster.
+* Install the {product-title} CLI (oc).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the web console, navigate to *Compute* -> *Bare Metal Hosts*.
+. Select *Add Host* -> *New with Dialog*.
+. Specify a unique name for the new bare metal host.
+. Set the boot mac address.
+. Optionally, enable power management for the host. This allows {product-title} to control the power state of the host.
+. Enter the user credentials for the host baseboard management controller (BMC) device.
+. Select to Power on the host after creation, and select *Create*.
+
+[NOTE]
+====
+This procedure creates the bare metal host. In order to create a corresponding bare metal node, scale up the number of replicas to match the number of available bare metal hosts using the `oc scale` command and the appropriate bare metal machine set.
```

Comment:
```suggestion
This procedure creates the bare metal host. In order to provision this host into a node, scale up the number of machine replicas in the appropriate machine set to match the number of bare metal hosts in the cluster. This can be done in the web console or using the `oc scale` command.
```

Something along these lines, providing at least a hint that it can be done through the UI as well since they chose to add the host via the UI already.

Reply:
I'll add this as a step in the procedure. It makes sense to document increasing the replica count from the UI.